editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2606,7 +2606,7 @@ void ThemeTypeEditor::_constant_item_changed(float p_value, String p_item_name)
 	edited_theme->set_constant(p_item_name, edited_type, int(p_value));
 }
 
-void ThemeTypeEditor::_edit_resource_item(RES p_resource) {
+void ThemeTypeEditor::_edit_resource_item(RES p_resource, bool p_edit) {
```

Comment:
Can you add the `edit` argument name in the `bind_method` call too? It's not needed but does surface in the GDNative API.

Reply:
Ah, I don't have a single method in this file that lists its parameters with `ClassDB::bind_method` because they are all internal. Should I add all of them?

Reply:
Ah no, if they don't appear in the json when you run `godot --gdnative-generate-json-api api.json`, no need to add them.

Reply:
Yeah neither class seems to be in the JSON so it's fine as is.